css/annotations/popup.scss

```diff
@@ -1,7 +1,8 @@
 // Colors for buttons.
 $button-background-color: #f7f7f7;
 $button-hover-color: #e6ebf5;
-
+$blue: #424bcd;
+$white: #fff;
```

Comment:
Use semantic variable names for theming (like the existing `$button-background-color` or `$button-hover-color`).  Imagine for example a designer styling the Stock Tools with a dark theme with orange highlight colors, then color variable names like $blue and $white are misleading.

Reply:
I was curious about the good practices for naming SCSS. Shall we add this to.. erm some guideline's readme? Not sure where could it fit nice

Reply:
Not sure either... But if you look at the naming in highcharts.scss it is generally semantic. The idea would be that it would make sense regardless of the actual color used.

Reply:
Descriptions like "light-color" and "dark-color" is also not good. In a typical dark mode design, the background would be dark and the typography light, so this would be counterintuitive. It is desireble to reduce the color palette in Highcharts, so I suggest we replace these color names _and_ values with the existing colors in `highcharts.scss`. We can for example use `$highlight-color-80` for the blue and `$background-color` for the white.